src/Domain/IndexManagement/IndexConfigurationRepository.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace JeroenG\Explorer\Domain\IndexManagement;
```

Comment:
I believe the repository belongs in Application?

Reply:
No I don't think so, the repository interface can be in the domain, while the implementation is infrastructure. 
for example: https://matthiasnoback.nl/2021/02/does-it-belong-in-the-application-or-domain-layer/
but it can be moved to the application if you like.